### DISCUSSION TITLE
Localization fallback

### DISCUSSION THREAD

**Discussion by Ravenna** [statamic/cms#5319] [Troubleshooting]
I have a site that is 90% English. The client has 2 pages they want to have translated.

I ran the multi-site generator and I see how I can use languages with this setup.  But I want the English pages to show if there is no translated page.  How would I set this up?

**Comment by Ravenna** [comment]
Bump?

**Comment by jasonvarga** [comment]
You can enable "entry propagation" on the collection.
This will essentially copy the English entry into other sites whenever you create the entry.

This only does it for new entries though, so you'll need to manually save the existing entries into the other sites.

You could write a little script or command to loop over your entries and do it automatically one time.

Something like this (untested)

```php
Entry::all() // assuming you want entries in all collections to work this way
  ->each(function ($entry) {
    Site::all()
      ->reject('en') // you already have english ones.
      ->each(fn ($site) => $entry->makeLocalization($site->handle())->save());
  });
```
